androidTEst/androidTEst/Screens/AllFilesScreen.cs

```diff
@@ -0,0 +1,116 @@
+﻿using androidTEst.Extension;
+using Aquality.Appium.Mobile.Elements.Interfaces;
+using Aquality.Appium.Mobile.Screens;
+using OpenQA.Selenium;
+
+namespace Aquality.Appium.Mobile.Template.Screens.Login
+{
+    public class AllFilesScreen : Screen
+    {
+        private readonly IButton AddNewFileButt = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/fab_main"), "AddNewFileButt");
```

Comment:
```suggestion
        private readonly IButton AddNewFileButton = ElementFactory.GetButton(By.Id("com.nextcloud.client:id/fab_main"), "Add New File");
```
чтобы не дублировать в логах, в имени элемента не указывается его тип, а вот в именни переменной указывается полностью